test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -1718,6 +1718,36 @@ def foo(
                 inputs={},
             ),
         ),
+        (
+            Circuit().rx(0, np.pi),
+            OpenQasmProgram(
+                source="\n".join(
+                    [
+                        "OPENQASM 3.0;",
+                        "bit[1] b;",
+                        "qubit[1] q;",
+                        "rx(π) q[0];",
+                        "b[0] = measure q[0];",
+                    ]
+                ),
+                inputs={},
+            ),
+        ),
+        (
+            Circuit().rx(0, 2 * np.pi),
+            OpenQasmProgram(
+                source="\n".join(
+                    [
+                        "OPENQASM 3.0;",
+                        "bit[1] b;",
+                        "qubit[1] q;",
+                        "rx(τ) q[0];",
```

Comment:
Honestly, might as well test all three built-ins

Reply:
I think it's overkill to test all 6. I test pi and tau since pi is a constant and tau is a formula (2 * pi), and not both of them worked the same way.